src/linking/remove_internal_symbols.cpp

```diff
@@ -23,6 +23,10 @@ Author: Daniel Kroening
 
 #include "static_lifetime_init.h"
 
+static std::unordered_set<irep_idt> loop_contracts_subs{
+  ID_C_spec_loop_invariant,
+  ID_C_spec_decreases};
```

Comment:
Please a void an object of static lifetime here. Just use `{ID_C_spec_loop_invariant, ID_C_spec_decreases}` in place.